From a6f39a12d7fef85c8ce40bc3898b17adf31bc2ba Mon Sep 17 00:00:00 2001
From: Adel Gadllah <adel.gadllah@gmail.com>
Date: Sun, 04 Aug 2013 09:35:35 +0000
Subject: overview: Reset opacity when not animating

We are not resetting the opacity when we are not animating, which can cause
a hidden window to end up with opacity 0 if we remove the tween to early.
---
diff --git a/js/ui/workspace.js b/js/ui/workspace.js
index e2e461d..2c8ff22 100644
--- a/js/ui/workspace.js
+++ b/js/ui/workspace.js
@@ -1094,6 +1094,7 @@ const Workspace = new Lang.Class({
                 Tweener.removeTweens(clone.actor);
                 clone.actor.set_position(x, y);
                 clone.actor.set_scale(scale, scale);
+                clone.actor.set_opacity(255);
                 clone.overlay.relayout(false);
                 this._showWindowOverlay(clone, overlay, isOnCurrentWorkspace);
             }
--
cgit v0.9.2
